packages/synapse-react-client/src/components/SynapseHomepageV2/SynapseByTheNumbersItem.tsx

```diff
@@ -1,15 +1,86 @@
 import { Box, Typography } from '@mui/material'
-import React, { ReactElement } from 'react'
+import { Row } from '@sage-bionetworks/synapse-types'
+import Plotly, { Datum } from 'plotly.js-basic-dist'
+import React from 'react'
+import createPlotlyComponent from 'react-plotly.js/factory'
 
+const Plot = createPlotlyComponent(Plotly)
 export type SynapseByTheNumbersItemProps = {
   title: string
   description: string
-  plot: ReactElement
+  rows: Row[]
+  plotXColIndex: number
+  plotYColIndex: number
 }
 
 export const SynapseByTheNumbersItem: React.FunctionComponent<
   SynapseByTheNumbersItemProps
-> = ({ title, description, plot }) => {
+> = ({ title, description, rows, plotXColIndex, plotYColIndex }) => {
+  if (rows.length == 0) {
+    return <></>
+  }
+  const xArray: Datum[] = []
+  const yArray: Datum[] = []
+  for (const row of rows) {
+    for (let j = 1; j < row.values.length; j += 1) {
+      // create pairs of data
+      const rowValues = row.values
+      xArray.push(rowValues[plotXColIndex])
+      yArray.push(rowValues[plotYColIndex])
+    }
+  }
+  //convert table data into numbers
+  const yArrayNumbers = yArray.map(v => parseFloat(v!.toString()))
+
+  const plotData: Partial<Plotly.PlotData>[] = [
+    {
+      type: 'scatter',
+      mode: 'lines',
+      fill: 'tozeroy',
+      fillcolor: '#336663',
+      line: {
+        color: '#6ba89d',
+        shape: 'spline', //smooth line, will need to do a more complex cubic spline interpolation (introducing new values between the data points) if we want to really smooth this out
```

Comment:
it still doesn't look smooth.  My hope is that with more data, this will look better over time.  If we need to do something more complex (to generate points between the actual data), then I have some notes here.